bitnami/mysql/values-production.yaml

```diff
@@ -31,6 +31,14 @@ service:
   type: ClusterIP
   port: 3306
 
+## String to partially override mysql.fullname template (will maintain the release name)
```

Comment:
Place this section between `image` and `service` sections as you did in the values.yaml file